Tests/UnitTests/Purchasing/CustomerInfoTests.swift

```diff
@@ -818,12 +833,12 @@ class BasicCustomerInfoTests: TestCase {
 
     func testCopyWithNewRequestDateUpdatesEntitlements() throws {
         expect(self.customerInfo.entitlements.active).to(haveCount(2))
-        expect(self.customerInfo.entitlements["old_pro"]?.isActive) == false
```

Comment:
This test was added in #2310.
`old_pro` no longer works because of the new grace period, so I've added a new `expired_pro` that expired not too far in the past.